device_access_fn.cpp

```diff
@@ -1638,15 +1642,15 @@ bool parseAndSyncTime(Resource *r, ResourceItem *item, const deCONZ::ApsDataIndi
 /*! A generic function to read ZCL attributes.
     The item->readParameters() is expected to be an object (given in the device description file).
 
-    { "fn": "zcl", "ep": endpoint, "cl" : clusterId, "at": attributeId, "mf": manufacturerCode, "noseq": noSequenceNumber  }
+    { "fn": "zcl:attr", "ep": endpoint, "cl" : clusterId, "at": attributeId, "noseq": noSequenceNumber  }
 
-    - endpoint, 0xff means any endpoint
+    - endpoint, (optional) 0xff means any endpoint
```

Comment:
Ha interesting looks like a bug to me also in former version.

The endpoint is not optional, if it isn't specified it becomes 0xFF and the request goes into the void since we're doing unicasts. 0 is allowed, it's the `AutoEndpoint` which uses the endpoint from the items resource `.uiniqueid`